src/ActiveLogin.Authentication.BankId.AspNetCore.Qr/ActiveLogin.Authentication.BankId.AspNetCore.Qr.csproj

```diff
@@ -0,0 +1,57 @@
+﻿<Project Sdk="Microsoft.NET.Sdk">
+  <PropertyGroup>
+    <TargetFramework>netcoreapp3.0</TargetFramework>
+    <LangVersion>latest</LangVersion>
+    <NeutralLanguage>en</NeutralLanguage>
+    <NoWarn>1701;1702;1591;CS7035</NoWarn>
+    <TreatWarningsAsErrors>true</TreatWarningsAsErrors>
+
+    <AssemblyName>ActiveLogin.Authentication.BankId.AspNetCore.Qr</AssemblyName>
```

Comment:
I've been thinking about the name, and I think it should be:
`ActiveLogin.Authentication.BankId.AspNetCore.QRCoder`

That follow the convention Microsoft has on third party libs. We include the name of the lib we use.

Reply:
Project, folders, package etc should also be renamed.